File: Pascal/Lab1/Variant7.pas
Program lab1;
Uses Crt;
Var a,b,x,Formula: real;
Begin
  Write('Введите первое число: '); Readln(a);
  Write('Введите второе число: '); Readln(b);
  Write('Введите третье число: '); Readln(x);
  ClrScr;
  Formula:=sqrt(sqr(x)+b)-(sqr(b)*power(sin(x+a),3))/x;
  Writeln('Вариант 7.  ',Formula);
End.

File: Python/Lab1/Variant 7.py
from math import *
a = float(input("Введите значение a: "))
b = float(input("Введите значение b: "))
x = float(input("Введите значение x: "))
w = sqrt(x*x+b)-b*b*pow(sin(x+a),3)/x
print("\nЗначение w =", w)
